notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -862,7 +913,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.13"
+   "version": "3.12.10"
```

Comment:
We keep taking turns overwriting this in various PRs. I think we still have the requirement for `<= 3.11` for some pieces, right?